File: answer.py
import string
def DNA_strand(dna):
    return dna.translate(string.maketrans("ATCG","TAGC"))
  
pairs = {'A':'T','T':'A','C':'G','G':'C'}
def DNA_strand(dna):
    return ''.join([pairs[x] for x in dna])

def DNA_strand(dna):
    new_dna=""
    for i in dna:
        match i:
            case "A":
                i="T"
                new_dna+=i
            case "T":
                i="A"
                new_dna+=i
            case "C":
               i="G" 
               new_dna+=i
            case "G":
                i="C"
                new_dna+=i
    return new_dna

File: answer.js
function DNAStrand(dna) {
  return dna.replace(/./g, function(c) {
    return DNAStrand.pairs[c]
  })
}

DNAStrand.pairs = {
  A: 'T',
  T: 'A',
  C: 'G',
  G: 'C',
}

const DNAStrand= dna =>{
    return dna.split("").map(e=>{
        switch(e){
            case "A":
                e="T"
                break;
            case "T":
                e="A"
                break;
            case "G":
                e="C"
                break;
            case "C":
                e="G"
                break
        }
     return e
    }).join("")
}

function DNAStrand(dna){
  var obj = {"A":"T","T":"A","C":"G","G":"C"}
  return dna.replace(/[ATCG]/g,x => obj[x])
}
